api/src/types/api.ts

```diff
@@ -48,6 +47,17 @@ export interface Paper {
   citationVelocity?: number;
 }
 
+export interface PaperWithEntityCounts extends PaperIdWithEntityCounts {
```

Comment:
I feel like there's a better way to name this/these...but 🤷

Reply:
Yea, totally agree. I think some cleanup of the types is definitely warranted.